File: El Mamoun/Lv 1/Problem 13.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

/*
  Problem 13:Write a program to ask the user to enter 3 numbers:
 • A
 • B
 • C
 Then Print Max number
*/

namespace Problem_13
{
    internal class Program
    {
        static int ReadNumber()
        {
            int Num;
            Console.Write("Please Enter a Number? ");
            Num = Convert.ToInt32(Console.ReadLine());

            return Num;
        }
        static int Max3Num(int Num1, int Num2,int Num3)
        {
            return (Num1 > Num2) ? (Num1 > Num3) ? Num1 : (Num2 > Num3) ? Num2 : Num3 : (Num3 > Num2) ? Num3 : Num2;
        }
        static void PrintResult(int Max)
        {
            Console.Write(Max);
        }


        static void Main(string[] args)
        {
            PrintResult(Max3Num(ReadNumber(), ReadNumber(), ReadNumber()));

            Console.ReadKey();
        }
    }
}

File: El Mamoun/Lv 1/Problem 13.cpp
#include<iostream>
using namespace std;

/*
  Problem 13:Write a program to ask the user to enter 3 numbers:
 • A
 • B
 • C
 Then Print Max number
*/

int ReadNumber()
{
	int number = 0;

	cout << "Please Enter a Number? ";
	cin >> number;

	return number;
}

int Max3Number(int Num1, int Num2,int Num3)
{

	if (Num1 > Num2 && Num1 > Num3)
		return Num1;

	else if (Num2 > Num1 && Num2 > Num3)
		return Num2;
	else
		return Num3;
}

void PrintMaxNumber(int Max)
{
	cout << "\nMax Number is = " << Max << endl;
}

int main()
{

	int Num1, Num2, Num3;

	Num1 = ReadNumber();
	Num2 = ReadNumber();
	Num3 = ReadNumber();

	PrintMaxNumber(Max3Number(Num1, Num2, Num3));

	return 0;
}